common/templates/layouts/collection.njk

```diff
@@ -6,36 +6,54 @@
   }) }}'});
 {% endblock %}
 
-{% if period === 'week' %}
-  {% set datetime = dateRange | formatISOWeek %}
-  {% set displayDate = dateRange | formatDateRangeAsRelativeWeek %}
-{% else %}
-  {% set datetime = dateRange | first %}
-  {% set displayDate = datetime | formatDateAsRelativeDay %}
-{% endif %}
-
 {% block pageTitle %}
-  {{ t("collections::page_title_with_date", {
-    context: context,
-    date: displayDate | lower
-  }) }}
+  {% if period === 'week' %}
+    {{ t("collections::page_title_with_date", {
+      context: context,
+      date: dateRange | formatDateRangeAsRelativeWeek
+    }) }}
+  {% else %}
+    {{ t("collections::page_title_with_date", {
+      context: context,
+      date: datetime | formatDateAsRelativeDay
+    }) }}
+  {% endif %}
 {% endblock %}
 
 {% block content %}
 
   {% block pageHeader %}
     <header class="govuk-grid-row govuk-!-margin-bottom-7">
       <div class="govuk-grid-column-two-thirds">
-        <span class="govuk-caption-xl">
+        <h1 class="govuk-heading-xl govuk-!-margin-bottom-2">
+          <span class="govuk-caption-xl">
           {{ t("collections::subheading", {
             context: context
           }) }}
-        </span>
-        <h1 class="govuk-heading-xl govuk-!-margin-bottom-2">
-          {{ appTime({
-            datetime: datetime,
-            text: displayDate
-          }) }}
+          </span>
+
+          <span class="govuk-!-display-block">
+          {% if period === 'week' %}
+            {{ dateRange | formatDateRange }}
+            {% if displayRelativeDate %}
+              {{ appTime({
+                datetime: dateRange | formatISOWeek,
+                text: '(' + dateRange | formatDateRangeAsRelativeWeek | lower + ')',
+                classes: 'app-secondary-text-colour'
+                }) }}
+            {% endif %}
+          {% else %}
+            {% set datetime = dateRange | first %}
+            {{ datetime | formatDateWithDay }}
+            {% if displayRelativeDate %}
+              {{ appTime({
+                datetime: datetime,
+                text: '(' + datetime | formatDateAsRelativeDay(' ') | lower + ')',
```

Comment:
What's the purpose of passing the empty space as the argument to `formatDateAsRelativeDay`? Is it to prevent it from displaying if not a relative date? That feels a little hard to read if so, wonder if there is a better way.

It would also be good to remove the lowercasing to keep consistency with other uses of the relative date:

<kbd><img src="https://user-images.githubusercontent.com/3327997/89767638-a4d01180-dafa-11ea-89b2-de68086c6fd8.png"></kbd>

Reply:
Yes, the code breaks with an empty space and displays nothing.
That chunk of code requires some love as the empty space is not the only _ugly hack_ that comes to my mind.
Concatenating parenthesis, with a `formatDate` and `lower` suggest that there is quite a lot of logic being executed.

This goes back to the previous comment:

1. I can introduce a new filter that doesn't display the date if it is not {Today,Tomorrow,Yesterday} 
1. I can edit the macro and move the code there.

As for the upper case, in one of the last Government project, I've learnt that most words should not be capitalised for the sake of it as they are more difficult to read. I'm not sure if there's a GovUK toolkit equivalent for content design, but this includes some guidance on capitalisation https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style

However if the rest of the website uses the capital T, I will change the rest of the code to be aligned to that.

Reply:
>I can introduce a new filter that doesn't display the date if it is not {Today,Tomorrow,Yesterday}

We already have a filter which could be re-used for this purpose: [`formatDateWithRelativeDay`](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/73c90bdf70fc5c965d9c0e0d99612c7d8c7ad617/config/nunjucks/filters.js#L194). Granted this is only for a single date but this could be replicated to do the same for relative week display.

> As for the upper case, in one of the last Government project, I've learnt that most words should not be capitalised for the sake of it as they are more difficult to read. I'm not sure if there's a GovUK toolkit equivalent for content design, but this includes some guidance on capitalisation https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style
>
> However if the rest of the website uses the capital T, I will change the rest of the code to be aligned to that.

I had another look through here and found a [section on dates](https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#dates) which uses lowercase as you suggested. It does however, use a relative date first, then the actual date in brackets. But let's stick with the way round we have it for now but can we change both instances to be consistent?

If you do re-use that filter then you could lowercase in in the filter once. I pushed [a branch](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/commit/8008f87bf28ad6a3a50c8c8171617cfde952ad0f) which re-used the filter as an example. I think it would greatly reduce the logic around this change but I also understand this change has been pending a while now so I'll leave the decision up to you. 

I do think it would be good to remain consistent with date display and to avoid any knock-on regressions in page titles with this change.

Reply:
Amended for consistent capitalisation.